MyServlet/src/main/webapp/Login/LoginForm.jsp

```diff
@@ -0,0 +1,22 @@
+<html>
+<head>
+<meta charset="UTF-8">
+<title>Login Page</title>
+</head>
+<body>
+  <form action = "login" method = "post">
+    <table>
+      <tr>
```

Comment:
I think u can merge this jsp with `loginFail.jsp` using jsp condition statements and passing data from servlet to jsp
https://stackoverflow.com/questions/5935892/if-else-within-jsp-or-jstl
https://www.programmergate.com/pass-data-servlet-jsp/
This action will help us keep a small amount of necessary jsp files.